src/hotspot/share/gc/shenandoah/shenandoahFreeSet.hpp

```diff
@@ -230,8 +401,25 @@ class ShenandoahFreeSet : public CHeapObj<mtGC> {
 
   void print_on(outputStream* out) const;
 
+  // This function places all regions that have allocation capacity into the mutator partition, or if the region
+  // is already affiliated with old, into the old collector partition, identifying regions that have no allocation
+  // capacity as NotFree.  Subsequently, we will move some of the mutator regions into the collector and old collector
+  // partitions with the intent of packing old collector memory into the highest (far rightmost) addresses of the heap,
+  // young collector memory into higher address, and mutator memory consuming the lowest addresses of the heap.
```

Comment:
Is the part of the comment that starts at "Subsequently, we will ..." done as part of this method? If not, then it should go into the caller of this method (after this method is called), rather than stating what the caller will do. I'd keep documentation of these methods very simple: what its parms are and what it does.

Reply:
Thanks.  I've adjusted the comments as guided.